kubectl-rook-ceph.sh

```diff
@@ -244,12 +244,6 @@ function fetch_mon_endpoints() {
   KUBECTL_NS_CLUSTER get cm rook-ceph-mon-endpoints -o json | jq --monochrome-output '.data.data' | tr -d '"' | tr -d '=' | sed 's/[A-Za-z]*//g'
 }
 
-wait_for_deployment_to_be_running() {
```

Comment:
This function is needed by the restore-quorum command, so if we can't call this helper from the CI, it's a simple one to duplicate.

Reply:
But we can call this function from a github helper file,
```tests/github-action-helper.sh wait_for_deployment_to_be_running rook-ceph-tools "$ROOK_CLUSTER_NAMESPACE"```

So with the krew plugin you mean to say it won't be possible to call this way and it will fail as it might not have this directory with it??

Reply:
Correct, the krew plugin only has this one .sh file installed, so the helper script is not available.